src/test/java/com/oath/halodb/EstimatedHistogramTests.java

```diff
@@ -0,0 +1,121 @@
+package com.oath.halodb;
+
+import com.oath.halodb.histo.EstimatedHistogram;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.internal.junit.ArrayAsserts.assertArrayEquals;
```

Comment:
Why static import for assertArrayEquals but not other assert types?

Reply:
Changed